common-ui-compose/src/main/java/app/tivi/common/compose/PosterCard.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.tivi.common.compose
+
+import androidx.compose.foundation.Text
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Stack
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Card
+import androidx.compose.material.EmphasisAmbient
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.ProvideEmphasis
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import app.tivi.data.entities.TiviShow
+import app.tivi.data.entities.TmdbImageEntity
+import dev.chrisbanes.accompanist.coil.CoilImageWithCrossfade
+
+@Composable
+fun PosterCard(
+    show: TiviShow,
+    poster: TmdbImageEntity? = null,
+    onClick: (() -> Unit)? = null,
+    modifier: Modifier = Modifier
+) {
+    Card(modifier = modifier) {
+        Stack(
+            modifier = if (onClick != null) Modifier.clickable(onClick = onClick) else Modifier
+        ) {
+            ProvideEmphasis(EmphasisAmbient.current.medium) {
```

Comment:
So if the poster is faded in, this is on top of the text? If so can we omit the text when it's loaded? Does Coil expose info to achieve this?